.github/workflows/main.yaml

```diff
@@ -467,6 +470,12 @@ jobs:
       - name: Run dialyzer
         run: docker run -v $PWD/:/github otp '/github/scripts/run-dialyzer'
 
+      - name: Run Python
+        run: docker run -v $PWD/:/github otp 'python3 --version'
+
+      - name: Run Scancode
+        run: docker run -v $PWD/:/github otp 'wget https://github.com/aboutcode-org/scancode-toolkit/releases/download/v32.2.1/scancode-toolkit-v32.2.1_py3.10-linux.tar.gz && tar -xvzf /github/scancode-toolkit-v32.2.1_py3.10-linux.tar.gz /github/scancode-toolkit-v32.2.1 && /github/scancode-toolkit-v32.2.1/configure && /github/scancode-toolkit-v32.2.1/scancode --version && alias scancode=/github/scancode-toolkit-v32.2.1/scancode && /github/scripts/scan-code.escript scancode --file-or-dir "${{ needs.pack.outputs.added_files }}" --template-path /github/scripts/scan-code/template.txt'
```

Comment:
```suggestion
        run: docker run -v $PWD/:/github otp 'sudo apt-get install pip && pip install scan-code-toolkit && scancode --version && /github/scripts/scan-code.escript scancode --file-or-dir "${{ needs.pack.outputs.added_files }}" --template-path /github/scripts/scan-code/template.txt'
```

Lets try this

Reply:
Let's see :) 
I read that downloading something from Github needs either some permission or special flag, but I have yet found time to look into it further.

I guess we could place also pip when building the docker container, since the container is just for the Github build, isn't it?

Reply:
Yes, we can put pip (and scan-code) in the container if we want to, unless it increases the image size dramatically. The whole image needs to be downloaded by each job in this workflow so we don't want it to become unnessasarily large.